src/commons/XMLParser/XMLParserHelper.ts

```diff
@@ -202,6 +203,7 @@ const makeProgramming = (
     testcases: publicTestcases.map(testcase => makeTestcase(testcase)),
     testcasesPrivate: privateTestcases.map(testcase => makeTestcase(testcase)),
     answer: solution ? (solution[0] as string).trim() : '',
+    lastModifiedAt: '2023-08-05T17:48:24.000000Z',
```

Comment:
Why is there a magic string here?

Reply:
That field is not really important here. Should I use the date function instead?

Reply:
Replaced with the Date function.